packages/storybook/stories/ControlBar/Buttons/Participants/styles/ParticipantStyles.tsx

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { ParticipantsButtonStyles } from '@azure/communication-react';
+
+/**
+ * styles to make sure that the border on the participant menu isnt overriden buy the csutom styles example
```

Comment:
Is there an alternative way to fix the red border issue? If you hit "show code" on the button example this makes the snippet unable to copy paste (as they don't have this styles as an import), it also makes the snippet a little more confusing:
![image](https://user-images.githubusercontent.com/2684369/156098223-5e242de0-9b93-4298-b0f6-e90ab5c47570.png)

Reply:
I will keep digging into this, but near as I can tell when the custom styles version of the button is clicked it add the red border to all of the buttons. The red border is not there until after the custom styles button is selected, until that point all other participant lists render correctly.